apps/etterlatte-brev-api/build.gradle.kts

```diff
@@ -17,7 +17,10 @@ dependencies {
     implementation(project(":libs:etterlatte-migrering-model"))
     implementation(project(":libs:etterlatte-brev-model"))
     implementation(project(":libs:etterlatte-oppgave-model"))
-    implementation(project(":libs:rapidsandrivers-extras")) // TODO: skal bort
+    implementation(libs.ktor2.servercio) // Rart at denne må inn siden libbet også definerer denne
```

Comment:
Avhengnadar i gradle er ikkje transitive med `implementation`, så det er nok forklaringa. (Må bruke `api` for å få dei transitive, men det er fint å unngå)

Reply:
Det stemmer, pushet opp endringer på det nu!  Skal få til ei oppryddning her senere

Reply:
fixd